README.md

```diff
@@ -69,6 +69,45 @@ Default: `[ 'GET' ]`
 
 This property allows a user to pass the list of HTTP request methods accepted by the server. 
 
+### generateContent
+
+Type: `Function`  
+Default: `undefined`
+
+This option accepts a `Function` value which is called before the files being served are read from the filesystem. It can be used to modify the file on the fly, skip their processing altogether or simply log their generation. For instance, the following configuration will replace the string `<!--self generated vars-->` with `<script type="text/javascript">var global_request = { method: "GET" }</script>` in every .html file requested via HTTP GET.
+
+```js
```

Comment:
This example is definitely too complicated for description :)

Reply:
Heh, I agree, I used the actual use case I'm proposing this function tho. It could be changed to the one used in the test suite (a simpler replace).

Here is the real world use case this would solve (and the whole motivation for this PR): as of today here is no way to detect if the request was a POST and, now that POSTs are accepted at all (due to the `methods` recently accepted option), there is no way to actually pass the post data to the scripts.

The example contained in the readme does just that, but partially. Parse the POST body and add to the global_request var and it becomes possible to inform the scripts what the actual posted data was.

That's an overkill for the README, I agree, should I change it to the "My" -> "Modified" example I included in the tests?